openapi_python_client/parser/properties/model_property.py

```diff
@@ -20,6 +20,7 @@ class ModelProperty(Property):
     _json_type_string: ClassVar[str] = "Dict[str, Any]"
 
     template: ClassVar[str] = "model_property.pyi"
+    is_dict: ClassVar[bool] = True
```

Comment:
not that happy with this, but couldn't think of a better way

Reply:
We might want a more broad type here in the future, as arrays and other might need more special handling but honestly this is "good enough" for now.

Reply:
What is this for, to tell if the property is a `ModelProperty`? 


Other places on the codegen check if `property.template` is set to tell if it's a templated (modeled) property (as opposed to a primitive one). Maybe something similar could be used here?

Reply:
Yeah. Right now it only applies to `ModelProperties`, not any other type of property.